h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -130,7 +149,7 @@ else if (line.equals("[columns]")) {
       } else if (section == 1) {
         // [info] section: just parse key-value pairs and store them into the `info` map.
         String[] res = line.split("\\s*=\\s*", 2);
-        info.put(res[0], res[0].equals("uuid")? res[1] : ParseUtils.tryParse(res[1]));
+        info.put(res[0], res[0].equals("uuid")? res[1] : new RawValue(res[1]));
```

Comment:
Could you please explain your motivation for switching to RawValue? Were there any values that ParseUtils could not parse properly? It seems to be adding lots of complexity, and I'm not sure what the benefits are...